htmresearch/frameworks/pytorch/mnist_sparse_experiment.py

```diff
@@ -393,10 +396,17 @@ def runNoiseTests(self, params):
 
   def pruneWeights(self, params):
     """
-    TODO: Prune the weights whose absolute magnitude is <= params["minWeight"]
+    Prune the weights whose absolute magnitude is <= params["min_weight"]
```

Comment:
I think this should say "< params["min_weight"]" 

In other words, it prunes stuff strictly less than min_weight, which is actually better. min_weight of 0 will have no effect.